third_party/terraform/tests/resource_storage_hmac_key_test.go

```diff
@@ -0,0 +1,68 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestStorageHmacKey_update(t *testing.T) {
+	t.Parallel()
+
+	saName := saName()
+	bucketName := testBucketName()
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckStorageHmacKeyDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testGoogleStorageHmacKeyBasic(saName, bucketName),
+			},
+			{
+				ResourceName:      "google_storage_hmac_key.default",
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
```suggestion
				ImportStateVerifyIgnore: []string{"secret"},
```

Since we can't find the value of `secret` on import, we'll need to tell the test we can't retrieve it.

Reply:
Sorry, meant to write that as adding the line and not replacing.

Reply:
got this one, but see comment above